src/pkg/cli/composeDown.go

```diff
@@ -23,9 +25,34 @@ func ComposeDown(ctx context.Context, loader client.Loader, client client.Fabric
 		return "", ErrDryRun
 	}
 
+	accountInfo, err := provider.AccountInfo(ctx)
+	if err != nil {
+		return "", err
+	}
+
 	if len(names) == 0 {
 		// If no names are provided, destroy the entire project
-		return provider.Destroy(ctx, &defangv1.DestroyRequest{Project: projectName})
+		etag, err := provider.Destroy(ctx, &defangv1.DestroyRequest{Project: projectName})
+		if err != nil {
+			return "", err
+		}
+
+		err = client.PutDeployment(ctx, &defangv1.PutDeploymentRequest{
+			Deployment: &defangv1.Deployment{
+				Action:            defangv1.DeploymentAction_DEPLOYMENT_ACTION_DOWN,
```

Comment:
Couuld you add the corresponding action to the `composeUp` command please? I don't want to have to assume that `nil` is "UP" if we can avoid it.